src/App.tsx

```diff
@@ -1,27 +1,101 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
 
 export const App = () => {
+  const [todos, setTodos] = useState(todosFromServer);
+  const [title, setTitle] = useState('');
+  const [selectedUser, setSelectedUser] = useState('0');
+  const [titleError, setTitleError] = useState('');
+  const [userError, setUserError] = useState('');
+  const handleAllTodo = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    let hasError = false;
+
+    if (title.trim() === '') {
+      setTitleError('Please enter a title');
+      hasError = true;
+    } else {
+      setTitleError('');
+    }
+
+    if (selectedUser === '0') {
+      setUserError('Please choose a user');
+      hasError = true;
+    } else {
+      setUserError('');
+    }
+
+    if (!hasError) {
+      const newTodo = {
+        id: todos.length ? Math.max(...todos.map(todo => todo.id)) + 1 : 1,
```

Comment:
The logic for generating a new todo ID assumes that the IDs are sequential and unique. If there are any deletions or non-sequential IDs, this could lead to duplicate IDs. Consider using a more robust method for generating unique IDs, such as a UUID library.

Reply:
??? idk about UUID library. maybe, this is not my level??